Trimble.Modus.Components/Controls/TMAlert/TMAlert.xaml

```diff
@@ -0,0 +1,105 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentView xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Trimble.Modus.Components.TMAlert"
+             xmlns:modus="clr-namespace:Trimble.Modus.Components"
+             xmlns:local="clr-namespace:Trimble.Modus.Components.Controls.TMAlert"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
+             x:Name="alertView">
+  <ContentView.Resources>
+    <ResourceDictionary>
+      <ResourceDictionary.MergedDictionaries>
+        <local:TMAlertStyles/>
+      </ResourceDictionary.MergedDictionaries>
+      <converters:InverseBoolConverter x:Key="InverseBoolConverter" />
+      <converters:StringNotNullOrEmptyBoolConverter x:Key="StringNotNullOrEmptyBoolConverter"/>
+    </ResourceDictionary>
+  </ContentView.Resources>
+
+  <StackLayout x:Name="alertLayout"
+               HorizontalOptions="Fill"
+               VerticalOptions="Fill"
+               Margin="16">
+    <Border x:Name="contentBorder"
+            VerticalOptions="CenterAndExpand"
+            HorizontalOptions="FillAndExpand"
+            BindingContext="{x:Reference alertView}"
+            StrokeThickness="1"
+            Margin="0"
+            Padding="0"
+            HeightRequest="56">
+      <Border.StrokeShape>
+        <RoundRectangle CornerRadius="2" />
+      </Border.StrokeShape>
+
+      <Grid VerticalOptions="FillAndExpand"
+            HorizontalOptions="FillAndExpand"
+            ColumnSpacing="0"
+            Margin="0"
+            Padding="0">
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="Auto"/>
+          <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+
+        <BoxView x:Name="VerticalSeparator"
```

Comment:
If you look closely there's a spacing in left and top of the separator.

<img width="777" alt="Screen Shot 2023-12-01 at 3 56 51 PM" src="https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/0d022f65-a1f3-46f5-a6eb-342f6c2e1704">